chart/values.yaml

```
@@ -5,6 +5,16 @@ image:
   repo: openebs
   # -- Release tag for our images
   tag: develop
+  repoTags:
+    # Note: Below image tag configuration is optional and typically should never be
+    #       used. Setting specific image tags for the different repositories proves useful 
+    #       for some integration testing scenarios. Use the 'tag' option above to set
+    #       released/pre-release container image tags.
+    #       The below tag values  will be picked for images by default.
```

```suggestion
    #       The below tag values will be picked for images by default.
```
nit: Extra whitespace character

---

Removed extra whitespaces